pkg/cmd/roachtest/tests/decommission.go

```diff
@@ -363,10 +363,6 @@ func runDecommission(
 				return err
 			}
 
-			db := c.Conn(ctx, l, pinnedNode)
```

Comment:
I was wondering if this was there in the first place for some side-effecting, but it looks like a copy/paste typo in a rather old refactoring commit [1].

[1] https://github.com/cockroachdb/cockroach/commit/b028e5d2d27ed8e78390ada336238c1b62070d05